ScoutSuite/providers/aws/services/ec2/snapshots.py

```diff
@@ -0,0 +1,18 @@
+from ScoutSuite.providers.aws.resources.resources import AWSSimpleResources
+from ScoutSuite.providers.aws.facade.facade import AWSFacade
+from opinel.utils.aws import get_name
+
+
+class Snapshots(AWSSimpleResources):
+    async def get_resources_from_api(self):
+        return self.facade.ec2.get_snapshots(self.scope['region'], self.scope['owner_id'])
+
+    def parse_resource(self, raw_snapshot):
+        raw_snapshot['id'] = raw_snapshot.pop('SnapshotId')
+        raw_snapshot['name'] = get_name(raw_snapshot, raw_snapshot, 'id')
+        raw_snapshot['public'] = self._is_public(raw_snapshot)
+        return raw_snapshot['id'], raw_snapshot
```

Comment:
I'm a bit confused here, do you really need to store `raw_snapshot` instead of only keeping its id, name and public status?

Reply:
Yes, the rest of the info is relevant, as it is used in rules and is displayed in the report.